db/migrate/20240604150237_add_brain_modality_columns_to_big_query.rb

```diff
@@ -0,0 +1,22 @@
+class AddBrainModalityColumnsToBigQuery < Mongoid::Migration
+  # note this operation is safe to run even if your BQ table already has the given column
+  def self.up
+    columns = %w[has_morphology has_electrophysiology]
+    client = BigQueryClient.new.client
+    [CellMetadatum::BIGQUERY_DATASET, 'cell_metadata_test'].each do |dataset_name|
+      dataset = client.dataset(dataset_name)
+      if dataset.present? # ensure test dataset exists to avoid migration failure
+        table = dataset.table(CellMetadatum::BIGQUERY_TABLE)
+        columns.each do |column_name|
+          table.schema {|s| s.boolean(column_name, mode: :nullable)}
+        end
+      end
+    end
+  end
+
+  def self.down
+    # BigQuery does not support column deletions except by dropping and recreating the table,
+    # so we'll cross that bridge when/if we come to it
```

Comment:
Hopefully, we can get by with making columns optional ;)

Reply:
Ha yeah this is literally a copy/paste from previous migrations.  This might in fact have changed if the column is empty.